app/services/pledge_form_generator.rb

```diff
@@ -33,6 +33,10 @@ def patient_identifier
     patient.identifier
   end
 
+  def patient_random_pin
+    "[#{patient.random_pin}]"
```

Comment:
Sounds good to me! Did it like this to avoid having to conditionally render the [ ] => maybe I'll move this to being done at the point of pledge generation then so it shows for all new ones after merging!